### PR TITLE
Add alarm store

### PR DESCRIPTION
**Pull request by leechwin**
use json file

### CODE REVIEW

File: scripts/testhubot.coffee [hunk 1]
```diff
@@ -182,6 +195,46 @@ module.exports = (robot) ->
...
+    readJSONFile (err, data) ->
+      if err
+        console.log err
+      else
+        tempData = []
+        data.forEach (obj) ->
+          tempData.push obj  if obj.msg isnt msg
+        writeJSONFile tempData
+
+  # 알람초기화. data.json 파일의 내용을 모두 알람으로 등록
```
Comment (cimfalab) [inline scripts/testhubot.coffee:219]: 알람 초기화. data.json 파일의 내용을 모두 알람으로 등록.

File: scripts/testhubot.coffee [hunk 2]
```diff
@@ -182,6 +195,46 @@ module.exports = (robot) ->
...
+  init = ->
+    readJSONFile (err, data) ->
+      if err
+        console.log err
+      else
+        data.forEach (obj) ->
+          beforeMin = 30
```
Comment (cimfalab) [inline scripts/testhubot.coffee:226]: Move above loop

File: scripts/testhubot.coffee [hunk 3]
```diff
@@ -182,6 +195,46 @@ module.exports = (robot) ->
...
+          cronDate = new Date(obj.time)
+          cronDate.setMinutes cronDate.getMinutes() - beforeMin
+          CronJob = require("cron").CronJob
```
Comment (cimfalab) [inline scripts/testhubot.coffee:229]: Move above loop

Comment (leechwin) [inline scripts/testhubot.coffee:229]: Dear,
Coding style is extremely personal and everyone has their own preferred style.
You can fix it your own personal style.

Comment (cimfalab) [inline scripts/testhubot.coffee:229]: Dear,
It's not an issue with coding style :)
I think it's related with optimization and code readability.
Why do you repeatedly set the same value to beforeMin?
Why do you repeatedly require the same class CronJob?

But I will merge and modify by myself for your favor.
Thanks.

**Comment by cimfalab** [comment]
@leechwin Dear, can you fix the following?
Currently, bot always alarms to #general channel even if a user registers an alarm to #another channel.
It's nice if bot alarams to registered channel.
Thanks.

**Pull request merged**